gobblin-cluster/src/main/java/org/apache/gobblin/cluster/GobblinHelixMultiManager.java

```diff
@@ -363,12 +359,26 @@ void handleLeadershipChange(NotificationContext changeContext) {
     }
   }
 
+  /**
+   * Delete jobs from the helix cluster
+   */
+  @VisibleForTesting
+  public void cleanUpJobs() {
+    cleanUpJobs(this.jobClusterHelixManager);
+
+    if (this.taskDriverHelixManager.isPresent()) {
+      cleanUpJobs(this.taskDriverHelixManager.get());
+    }
+  }
+
   private void cleanUpJobs(HelixManager helixManager) {
     // Clean up existing jobs
     TaskDriver taskDriver = new TaskDriver(helixManager);
 
     Map<String, WorkflowConfig> workflows = taskDriver.getWorkflows();
 
+    log.debug("cleanUpJobs workflow count {} workflows {}", workflows.size(), workflows);
```

Comment:
Maybe just dump workflows.keySet() instead of the entire map?

Reply:
Sure.